android/src/main/java/co/nimblehq/ic/kmm/suv/android/ui/screens/home/HomeScreen.kt

```diff
@@ -30,40 +36,75 @@ fun HomeScreen(viewModel: HomeViewModel = getViewModel()) {
         )
     }
 
-    HomeScreenContent(currentDate, avatarUrlString, isLoading)
+    // TODO: Update survey data later
+    val uiModel = HomeUiModel(
+        HomeHeaderUiModel(
+            currentDate,
+            avatarUrlString,
```

Comment:
is this `imageUrl`?

Reply:
Fixed in https://github.com/suho/kmm-ic/pull/106/commits/029cfc14c4b123b651f9b3115660969719838097